### PR TITLE
54 app installer

### PR DESCRIPTION
**Pull request by nicolas-bastien**
This PR fix  : https://github.com/ezsystems/ezplatform-com/issues/54 

This PR should be merge after https://github.com/ezsystems/ezplatform-com/pull/60 as it use service declaration refactoring.

## Description

This PR provide an installer base on the demo code with data from ezplatform.
I added some documentation.

I added a ezplatform screenshot on the readme file + documentation

## Test

Manually by having a fresh install and run the installer command line.

## Evolution (?)

This PR provide a easy and clean installer but if we want to be really up to date with ezplatform.com maybe we can do another issue to directly download and unzip backup from ezplatform with a new option or another installer.

But I think this should be present as a backup or quick install in case dumps break or database and storage become too big.

+ @SylvainGuittard right now storage dump have a big 'original' folder (294,6 Mo) I guess it should be removed for archive ?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,4 +5,18 @@ eZPlatform.com - eZ Systems' Developer Hub for the Open Source PHP CMS eZ Platfo
 
 This repo contains the source files for the ezplatform.com website, built on eZ Platform and hosted by the eZ Systems team.
 
+![ezplatform.com](https://raw.githubusercontent.com/ezsystems/ezplatform-com/master/web/assets/doc/images/ezplatform.jpg)
```
Comment (clash82) [inline README.md]: 404

File: src/AppBundle/Installer/AppInstaller.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+use Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * Install data from ezplatform.com
+ */
+class AppInstaller extends CleanInstaller  implements Installer
+{
+    public function importData()
+    {
+        $this->runQueriesFromFile('app/Resources/installer/sql/app_data.sql');
```
Comment (clash82) [inline src/AppBundle/Installer/AppInstaller.php]: Nope, this have to be downloaded by the user first `https://ezplatform.com/dumps/ezplatform_page_db.sql.bz2`

File: src/AppBundle/Installer/AppInstaller.php [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    {
+        // configuration is included in ezplatform.yml file
+    }
+
+    public function importBinaries()
+    {
+        $this->output->writeln('Copying storage directory contents...');
+        $fs = new Filesystem();
+        $fs->mkdir('web/var/site');
+        $fs->mirror(
```
Comment (clash82) [inline src/AppBundle/Installer/AppInstaller.php]: Nope, `https://ezplatform.com/dumps/ezplatform_page_storage.tar.bz2`

File: README.md (hunk 1 above)
Comment (clash82) [inline README.md]: True, but you could also upload image on GH and then only link to it. We did something like this on ezplatformo-demo (https://github.com/ezsystems/ezplatform-demo/blob/master/README.md). Clever, huh? 😈

**Comment by nicolas-bastien** [comment]
@clash82 I updated the issue description, I'll work with another PR

**Pull request closed without merging**